tests/test_plotting.py

```diff
@@ -335,6 +335,29 @@ def test_IdealGas_plot_already_added():
         ValueError, match="Plot has already been added to this class instance"
     ):
         g.plot("v", "T")
+        
+def test_label_add_state():
+    """Test using a label in add_state."""
+    vd = VaporDome('water', ('v', 'T'))
+    st_1 = State('water', x =Q_(1.0,''), T= Q_(100, "degC"))
+    st_2 = State('water', x =Q_(0.0,''), T= Q_(100, "degC"))
+    assert(st_1._label == None)
```

Comment:
When comparing to _singletons_ (a fancy CS word meaning there's only ever one instance of a class; in Python `None` is a singleton), you want to use `is` for comparison.